src/index/Index.h

```diff
@@ -26,19 +29,23 @@
 #include "./TextMetaData.h"
 #include "./Vocabulary.h"
 
+using ad_utility::BufferedVector;
 using ad_utility::MmapVector;
+using ad_utility::MmapVectorView;
 using std::array;
 using std::string;
 using std::tuple;
 using std::vector;
 
 using json = nlohmann::json;
 
-using IdPairMMapVec = ad_utility::MmapVector<std::array<Id, 2>>;
 // a simple struct for better naming
 struct LinesAndWords {
```

Comment:
I think with the new paraeters `LinesAndWords` is not the right name for this stuct anymore. Maybe `VocabularyData`?